packages/link/__stories__/Link.stories.js

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
 import Link from '../src/Link';
+import React from 'react';
```

Comment:
i'm not sure how i feel about this... this is enforced by [sort-imports](https://eslint.org/docs/rules/sort-imports), which I think is useful to have, but i wish you could have exceptions because i like for react imports to come first in a file. the alternative would be to use the `allowSeparatedGroups` option, but then everyone would have to know to set up files like this:

```javascript
import React from 'react';

// other imports in separate group(s)
```

and i'm not sure i like that either

Reply:
Looks like there's this [plugin](https://github.com/import-js/eslint-plugin-import) which could grant us more control over import ordering? I do agree that it would be nice for React and other similar modules to be grouped separately but don't feel too strongly against the current sort-imports approach.

Reply:
ooo, no, you're right! i added that plugin for a different reason (it helps mitigate problems that can arise from using the ["hoisting" option in lerna](https://github.com/lerna/lerna/blob/main/doc/hoist.md#forgetting-dependencies)) and didn't realize we could enforce order with it. i'll update to use that instead -- thank you!

Reply:
i'll fix in #5 so i don't cause any conflicts